.circleci/cimodel/data/binary_build_data.py

```diff
@@ -62,7 +62,7 @@ def get_processor_arch_name(gpu_version):
     )),
     # Skip CUDA-9.2 builds on Windows
     windows=(
-        [v for v in dimensions.GPU_VERSIONS if v not in ['cuda92'] + dimensions.ROCM_VERSION_LABELS],
+        [v for v in dimensions.GPU_VERSIONS if v not in ['cuda92', 'cuda111'] + dimensions.ROCM_VERSION_LABELS],
```

Comment:
temporarily removing 11.1 for binary windows @peterjc123 @zasdfgbnm since Xiang is currently working on windows builder I believe

Reply:
Related PR: https://github.com/pytorch/builder/pull/574

Reply:
Just FYI, the PR is already merged.

Reply:
Yes, thanks :)